config/samples/druid_v1alpha1_etcd.yaml

```
@@ -62,6 +62,9 @@ spec:
   sharedConfig:
     autoCompactionMode: periodic
     autoCompactionRetention: "30m"
+  schedulingConstraints:
+    affinity: {}
+    topologySpreadConstraints: {}
```

```suggestion
    topologySpreadConstraints: []
```
Since `topologySpreadConstraints` is an array